### PR TITLE
System tests: stabilize css rewriting edge cases tests

### PR DESCRIPTION
**Pull request by oschaaf**
These tests sometimes flaked in the more extensive checkin tests

### CODE REVIEW

File: pagespeed/system/system_tests/css_minify_calc_function_value_zero.sh [hunk 1]
```diff
@@ -17,8 +17,8 @@
 
 start_test CSS minify with calc function and value 0.
 
-URL="$TEST_ROOT/css_minify_calc_function_value_zero.html?PageSpeedFilters=+inline_css"
-RESPONSE_OUT=$(http_proxy=$SECONDARY_HOSTNAME $WGET_DUMP $URL)
+URL="$TEST_ROOT/css_minify_calc_function_value_zero.html?PageSpeedFilters=rewrite_css"
+RESPONSE_OUT=$(http_proxy=$SECONDARY_HOSTNAME $WGET_DUMP --header 'X-PSA-Blocking-Rewrite: psatest' $URL)
```
Comment (jmarantz) [inline pagespeed/system/system_tests/css_minify_calc_function_value_zero.sh]: I had traditionally been avoiding using X-PSA-Blocking-Rewrite for de-flaking positive tests.  The main reason it's there is for negative tests, e.g. I expect a disallowed resource to never be optimized.

The reason I don't want to use that for positive tests is that it deviates from the production path.  And the mechanism we use to avoid the flakes in most cases is fetch_until.  Is there some reason that won't work here?

Comment (oschaaf) [reply pagespeed/system/system_tests/css_minify_calc_function_value_zero.sh]: @jmarantz `fetch_until` should work, and there's nothing that speaks for sticking with `X-PSA-Blocking-Rewrite` in this change. I'll amend this to use `fetch_until`

**Review by ashishk-1 — APPROVED** [review]

**Comment by oschaaf** [comment]
@jmarantz are you happy with the way this looks now?

**Review by jmarantz — APPROVED** [review]
Thanks Otto, looks perfect.
